text_prep/text_pre.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import pandas as pd
+import re
+import string
+import nltk
+from nltk.tokenize import word_tokenize
+from nltk.stem import WordNetLemmatizer 
+
+from nltk.corpus import stopwords
+nltk.download('stopwords')
+stop = stopwords.words('english')
+
+# project_dir = r'D:\GitHub\Projects\Analysis_of_Delivery_Companies_Reviews'
+# os.chdir(project_dir)
+
+df = pd.read_csv('dataframe.csv')
+
+def processing(df: pd.core.frame.DataFrame) :
```

Comment:
Could we create separate functions for each task ? For example, instead of a dataframe, we would like to process a string, which will correspond to a text/review. So, we would need something like a function removeNumbers(text), where text is a string and it will output the text without the numbers. Similar logic for other applications, like removing punctuation, etc

Reply:
okay, will make separate functions for each which will take text as input instead of dataframe.

Reply:
also, which branch do I need to commit to?

Reply:
I have created a branch called **issue/07_Text_Processing_Package** - lets use this to create the library. You can create a new folder called 'processing' and we can put the python file in there.